lib/js/jscore.nim

```diff
@@ -110,3 +110,15 @@ proc parse*(l: JsonLib, s: cstring): JsRoot {.importcpp.}
 since (1, 5):
   func debugger*() {.importjs: "debugger@".}
     ## https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/debugger
+
+  template jsexport*(symbol: untyped; alias = "") =
```

Comment:
please s/alias/name/ or similar, in future `alias` may become a keyword (refs https://github.com/nim-lang/Nim/pull/11992)